### PR TITLE
update file with retry

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- [update file with retry](https://github.com/airqo-platform/AirQo-frontend/commit/4149af670794638b3ba16f265e72ad633a11fa7c)

#### Status of maturity (all need to be checked before merging):
- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?
- Static code analysis

### CODE REVIEW

File: .github/workflows/deploy-frontends-to-production.yml [hunk 1]
```diff
@@ -90,16 +90,17 @@ jobs:
...
-          message: "Update netmanger production image tag to ${{ needs.image-tag.outputs.build_id }}"
-          createPR: false
+          action: fjogeleit/yaml-update-action@main # Action to retry
+          with: |
+            valueFile: "k8s/netmanager/values-prod.yaml"
+            propertyPath: "image.tag"
+            value: ${{ needs.image-tag.outputs.build_id }}
+            branch: deployment/argocd
+            message: "Update netmanger production image tag to ${{ needs.image-tag.outputs.build_id }}"
+            createPR: false
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontends-to-production.yml:103]: [update file with retry](https://github.com/airqo-platform/AirQo-frontend/pull/991/commits/4149af670794638b3ba16f265e72ad633a11fa7c)

Comment (Baalmart) [reply .github/workflows/deploy-frontends-to-production.yml:103]: Hi @123MwanjeMike , the link in your comment leads to this PR. Intentional?

**Review by Baalmart — COMMENTED** [review]
Hi @123MwanjeMike , please share more insights into this "retry"?
What are we solving here?

File: .github/workflows/deploy-frontends-to-production.yml (hunk 1 above)
Comment (123MwanjeMike) [reply .github/workflows/deploy-frontends-to-production.yml:103]: Hi @Baalmart 
It leads to commit 4149af670794638b3ba16f265e72ad633a11fa7c whose commit message is "update file with retry"

**Comment by 123MwanjeMike** [comment]
> Hi @123MwanjeMike , please share more insights into this "retry"? What are we solving here?

Hi @Baalmart, there was an issue with the parallelism where the update would fail due to difference in history caused by multiple commits being done at the same time. The retry part attempts to update the file one more time on fail. More on this problem can be found in [this issue](https://github.com/fjogeleit/yaml-update-action/issues/533)

**Review by Baalmart — APPROVED** [review]
Alright @123MwanjeMike ....thanks for this.

**Pull request merged**